README.md

```diff
@@ -475,6 +476,19 @@ These behave exactly like VueJs's transition directives, except they have differ
 
 ---
 
+### `x-ignore`
+**Example:**
+```html
+<div x-ignore="true">
+    <div x-data="{foo:'bar'}">
+        <span x-text="foo">"foo" will not set to 'bar' by Alpine</span>
```

Comment:
😂 I'm starting to feel bad about the reviews

```suggestion
        <span x-text="foo">"foo" will not be set to 'bar' by Alpine</span>
```

Reply:
Yeah, there might be a lot of these. :)